api/events.py

```diff
@@ -61,34 +51,24 @@ def search(lat, lon, radius=10, start_date="", end_date="", query="",
         ]
         query["query"]["bool"]["minimum_should_match"] = 1
 
-    if category:
-        query["query"]["bool"]["must"].append(
-            {"match": {"category": category}}
-        )
+    # if category:
```

Comment:
`category` ist ein paar zeilen hochgerutscht, nach `query["query"]["bool"]["should"]`